release_notes/ocp-4-20-release-notes.adoc

```diff
@@ -2402,17 +2402,22 @@ In the following tables, features are marked with the following statuses:
 +
 --
 ** In an {aws-short} private cluster, the load balancer becomes stuck in the `pending` state and reports the error: `Error syncing load balancer: failed to ensure load balancer: could not find any suitable subnets for creating the ELB`.
+* Starting with OCP 4.20, there is a decrease in the default maximum open files soft limit for containers. As a consequence, end users may experience application failures. To work around this problem, increase the container runtimes (CRI-O) ulimit configuration. (link:https://issues.redhat.com/browse/OCPBUGS-62095[OCPBUGS-62095])
```

Comment:
@kdrake20 Is this note in the correct place? Looking at it, I'm not sure how it relates to the bullet points either side of it or the bug that covers the overall bulleted list